app/src/androidTest/java/com/github/onlynotesswent/ui/EndToEndTest.kt

```diff
@@ -160,6 +179,37 @@ class EndToEndTest {
                   composable(Screen.EDIT_NOTE) {
                     EditNoteScreen(navigationActions, noteViewModel, userViewModel, fileViewModel)
                   }
+                  composable(Screen.FOLDER_CONTENTS) {
+                    FolderContentScreen(
+                        navigationActions, folderViewModel, noteViewModel, userViewModel)
+                  }
+                  composable(Screen.EDIT_MARKDOWN) {
+                    EditMarkdownScreen(
+                        navigationActions, noteViewModel, userViewModel, fileViewModel)
+                  }
+                }
+                navigation(
+                    startDestination = Screen.SEARCH_NOTE,
+                    route = Route.SEARCH,
+                ) {
+                  composable(Screen.SEARCH_NOTE) {
```

Comment:
rename this, it falls the CI, someone refactored it.

Reply:
forgot to add in the review but fix the missing dependencies on line 211 as well